cornflow-server/cornflow/tests/unit/test_cases.py

```diff
@@ -148,6 +148,7 @@ def test_new_case_execution(self):
         self.payload["solution"] = self.execution.data
         self.payload["solution_hash"] = self.execution.data_hash
         self.payload["user_id"] = self.user
+        self.payload["indicators"] = ""
 
         for key in self.response_items:
             self.assertEqual(self.payload[key], getattr(created_case, key))
```

Comment:
As `indicators` is not in `self.response_items` adding it to the `self.payload` doesn't do anything as it is not getting checked against the response of the endpoint. You are not testing if the change you made is actually valid or not.

Reply:
All done